### DISCUSSION TITLE
Add option for Mb/s along with MB/s

### DISCUSSION THREAD

**Discussion by Mitty** [sabnzbd/sabnzbd#2841] [Ideas]
Greetings,

What do y'all think about adding an option to display the download speed as Mb/s? This is a standard for network speeds where MB/s is used more for storage speeds. 

Thoughts?

Thanks!

**Comment by sanderjo** [comment]
Yes, it can be confusing for users

Adding an _option_ would be even more confusing: then in helpdesk forums you have to check which option the user is using. Now it's clear: always MB/s.

**Reply by Safihre** [reply]
Agreed. We already show the Internet Bandwith also in Mb/s as there indeed it's a common usage, but regular speeds in the application we want to keep fixed to MB/s.
